CTX.js

```diff
@@ -207,7 +161,7 @@ function contextObjectXMLToCOinS (text) {
 };
 
 function doImportFromText(text, showPrompt) {
-	var spans = [], items = [], zoteroItems = [];
+	var spans = [];
 	
 	// This is because we want to be able to read multiple such CTX elements in a single page
 	if (typeof text != "string" && text.length >= 1) {
```

Comment:
At this point, text is always going to be a string. Besides that, might as well just treat the input as XML and use `Zotero.getXML()` to get the parsed DOM tree, instead of reading everything in as text and then parsing to DOM in the translator.